tests/test_log_backend.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import unittest
+import mock
+
+from marquez_client.log_backend import LogBackend
+from marquez_client.utils import Utils
+
+log = logging.getLogger(__name__)
+
+
+class TestLogBackend(unittest.TestCase):
+    def setUp(self):
+        log.debug("TestLogBackend.setup(): ")
+        self.log_backend = LogBackend()
+        self.put_json = Utils.get_json("tests/resources/put.json")
+        self.post_json = Utils.get_json("tests/resources/post.json")
```

Comment:
for the logic of this test we really don't need those value to be real payloads. "PUT PAYLOAD" and "POST PAYLOAD" would have been fine.

Reply:
well. quality test data and reusable test data.